jicofo-common/src/main/kotlin/org/jitsi/jicofo/xmpp/muc/SourceInfo.kt

```diff
@@ -16,15 +16,31 @@
 package org.jitsi.jicofo.xmpp.muc
 
 import org.jitsi.jicofo.conference.source.VideoType
+import org.jitsi.utils.MediaType
+import org.jitsi.utils.logging2.LoggerImpl
 import org.json.simple.JSONObject
 import org.json.simple.parser.JSONParser
 
+private val logger = LoggerImpl(SourceInfo::javaClass.name)
+
 /** The information about a source contained in a jitsi-meet SourceInfo extension. */
 data class SourceInfo(
     val name: String,
     val muted: Boolean,
     val videoType: VideoType?
-)
+) {
+    val mediaType: MediaType? = name.indexOf('-').let { indexOfDash ->
```

Comment:
I'd rather we signal this explicitly rather than implicitly as here, though that can potentially be a separate PR (and would require client changes).